### DISCUSSION TITLE
How to edit the templates, the plausible-tracke

### DISCUSSION THREAD

**Discussion by yahuney** [plausible/analytics#4603] [Self-Hosted Support]
I have installed the plausible-tracker version. However, I can't find the directory to edit the templates (user interface, admin interface, etc.). Please help me. Sorry, this is my first time working with Docker.
Thank you so much,

**Comment by ruslandoga** [comment]
👋 @yahuney 

It’s not clear what you are asking. Would you be able to list the steps you have taken and what exactly you want to accomplish? What templates are you referring to?

**Reply by yahuney** [reply]
I have successfully installed plausible-tracker and am currently using it. But unlike a PHP website where I can simply find a file to edit the interface, here I can't find any relevant files despite searching everywhere. For example, I want to modify the homepage when people visit, change the welcome section, or remove the registration part (I want to invite friends only). As for the user interface, I also want to remove the footer section...

**Reply by ruslandoga** [reply]
UI configurations are not possible.

**Reply by ruslandoga** [reply]
You can configure Plausible to allow invites by setting https://github.com/plausible/community-edition/tree/v2.1.1#disable_registration to `invite_only`

**Reply by yahuney** [reply]
Thank you for your reply. I don't understand Docker, so I have a few more questions. Can I install https://github.com/plausible/analytics? If so, can I edit the template in /lib/plausible_web, right?

**Reply by ruslandoga** [reply]
Yes, you can. You can follow the commands from [Dockerfile.](https://github.com/plausible/analytics/blob/master/Dockerfile) But it might end up being too complicated. Using Docker is the easiest option available right now.
